packages/eas-cli/src/utils/__tests__/code-signing-test.ts

```diff
@@ -97,10 +97,10 @@ describe(parseMultipartMixedResponseAsync, () => {
     const form = generateMultipartBody(JSON.stringify({ hello: 'world' }));
 
     const parts = await parseMultipartMixedResponseAsync({
-      rawBody: form.getBuffer(),
-      headers: {
+      arrayBuffer: () => Promise.resolve(form.getBuffer()),
```

Comment:
```suggestion
      arrayBuffer: async () => form.getBuffer(),
```
I think we generally opt towards this syntax in expo (based on recent PR feedback I've been getting). I have no preference.